runtime/protocol/http-client-engines/http-client-engine-okhttp/jvm/src/aws/smithy/kotlin/runtime/http/engine/okhttp/OkHttpUtils.kt

```diff
@@ -159,11 +158,7 @@ internal class OkHttpProxySelector(
             else -> emptyList()
         }
     }
-
-    override fun connectFailed(uri: URI?, sa: SocketAddress?, ioe: IOException?) {
-        val logger = Logger.getLogger<OkHttpProxySelector>()
-        logger.error { "failed to connect to proxy: uri=$uri; socketAddress: $sa; exception: $ioe" }
-    }
+    override fun connectFailed(uri: URI?, sa: SocketAddress?, ioe: IOException?) {}
```

Comment:
**Question**: Do we intend to re-enable logging here once a default Slf4j binding is available? If so, I suggest leaving a `// FIXME` and possibly the commented code too for a starting place. As it is, this method looks pointless and like precisely the kind of thing I would delete if I came across it.